src/Doctrine/EntityClassGenerator.php

```diff
@@ -31,7 +31,7 @@ public function __construct(Generator $generator, DoctrineHelper $doctrineHelper
         $this->doctrineHelper = $doctrineHelper;
     }
 
-    public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $apiResource, bool $withPasswordUpgrade = false): string
+    public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $apiResource, bool $withPasswordUpgrade = false, bool $avoidDefaultComments = false): string
```

Comment:
Now that I see this, it might be better to have the last argument be `bool $generateRepositoryClass = true`. Then, we would only generate the repository class automatically if this is true. In `make:reset-password`, we would set this to `false`. Then we would call `generateRepositoryClass()` directly. It just feels more clear to do the 2 things independently... and maybe I should have always made this method *only* generate the entity class and not also the repository.

Reply:
Done